app/src/main/java/fr/free/nrw/commons/CommonsApplication.java

```diff
@@ -110,6 +110,10 @@ public class CommonsApplication extends MultiDexApplication {
 
     public static final String FEEDBACK_EMAIL_SUBJECT = "Commons Android App Feedback";
 
+    public static final String REPORT_EMAIL = "commons-app-android@googlegroups.com";
```

Comment:
I added this duplicate line in case we want to have a different email for reports in future.

Reply:
commons-app-android-private@googlegroups.com (private) might be better for instance in bullying cases.

Reply:
@nicolas-raoul Updated :)